app/vmstorage/main.go

```diff
@@ -374,6 +376,22 @@ func RequestHandler(w http.ResponseWriter, r *http.Request) bool {
 	}
 }
 
+func deleteStaleSnapshots() {
```

Comment:
I think, that this function must be added to the storage and included into storage watchers, like  freeDiskSpaceWatcher for example:
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/lib/storage/storage.go#L756